### PR TITLE
Add instructions when no results in the UI

### PR DESCRIPTION
**Pull request by juli1**
### What does this PR do? What is the motivation?

We had multiple customers blocked with no results in the UI. We explain what to do when they face this error.

### Merge instructions

Please merge after review.

### CODE REVIEW

File: content/en/code_analysis/faq/_index.md [hunk 1]
```diff
@@ -15,12 +15,28 @@ further_reading:
...
 
 ## Common questions
 
+### What to check if I see no results in the Datadog UI?
+
+If you are using GitHub, please check you use the GitHub App and have the required permissions.
+
+If you are using a Git provider other than GitHub, please first push results on your default branch. Your default
+branch should be one of the name along `master`, `main`, `prod`, `production` or `develop`. If your default
+branch is not one of those branches, you need to manually define it in the repository settings.
```
Comment (dastrong) [inline content/en/code_analysis/faq/_index.md:30]: These are on separate lines when they should be all on the same one

File: content/en/code_analysis/faq/_index.md [hunk 2]
```diff
@@ -15,12 +15,28 @@ further_reading:
...
+
+If you run the tool within your CI pipeline, please make sure that `datadog-ci` runs **at the root** of the repository
+being analyzed.
```
Comment (dastrong) [inline content/en/code_analysis/faq/_index.md:33]: Same thing about being on a separate line

File: content/en/code_analysis/faq/_index.md [hunk 3]
```diff
@@ -15,12 +15,28 @@ further_reading:
...
+
+### I get a 403 error when running the analyzer
+
+Please make sure you correctly specify the variables `DD_APP_KEY`, `DD_API_KEY` and `DD_SITE` when running the
+analyzer and `datadog-ci`.
```
Comment (dastrong) [inline content/en/code_analysis/faq/_index.md:38]: same comment as above

Comment (juli1) [reply content/en/code_analysis/faq/_index.md:38]: Not an issue with Markdown.

File: content/en/code_analysis/faq/_index.md (hunk 1 above)
Comment (juli1) [reply content/en/code_analysis/faq/_index.md:30]: Not an issue with Markdown.

File: content/en/code_analysis/faq/_index.md [hunk 4]
```diff
@@ -15,12 +15,28 @@ further_reading:
...
 ## Overview
 
-[Datadog Code Analysis][10] helps you identify and address issues in your code with Static Analysis (SAST) and Software Composition Analysis (SCA). 
+[Datadog Code Analysis][10] helps you identify and address issues in your code with Static Analysis (SAST) and Software Composition Analysis (SCA).
 
 This page provides a non-exhaustive list of frequently asked questions about Code Analysis.
 
 ## Common questions
 
+### What to check if I see no results in the Datadog UI?
```
Comment (estherk15) [inline content/en/code_analysis/faq/_index.md:24]:
```suggestion
### What to check if there are no results in the Datadog UI
```

File: content/en/code_analysis/faq/_index.md [hunk 5]
```diff
@@ -15,12 +15,28 @@ further_reading:
...
+
+If you are using GitHub, please check you use the GitHub App and have the required permissions.
+
+If you are using a Git provider other than GitHub, please first push results on your default branch. Your default
```
Comment (estherk15) [inline content/en/code_analysis/faq/_index.md:28]:
```suggestion
If you are using GitHub, check the GitHub App and make sure you have the required permissions.

If you are using a Git provider other than GitHub, first push results on your default branch. Your default
```

File: content/en/code_analysis/faq/_index.md [hunk 6]
```diff
@@ -15,12 +15,28 @@ further_reading:
...
+branch should be one of the name along `master`, `main`, `prod`, `production` or `develop`. If your default
+branch is not one of those branches, you need to manually define it in the repository settings.
+
+If you run the tool within your CI pipeline, please make sure that `datadog-ci` runs **at the root** of the repository
```
Comment (estherk15) [inline content/en/code_analysis/faq/_index.md:32]:
```suggestion
If you run the tool within your CI pipeline, make sure that `datadog-ci` runs **at the root** of the repository
```

File: content/en/code_analysis/faq/_index.md [hunk 7]
```diff
@@ -15,12 +15,28 @@ further_reading:
...
+being analyzed.
+
+### I get a 403 error when running the analyzer
```
Comment (estherk15) [inline content/en/code_analysis/faq/_index.md:35]:
```suggestion
### Getting a 403 error when running the analyzer
```

File: content/en/code_analysis/faq/_index.md (hunk 3 above)
Comment (estherk15) [inline content/en/code_analysis/faq/_index.md:38]:
```suggestion
Make sure you correctly specify the variables `DD_APP_KEY`, `DD_API_KEY` and `DD_SITE` when running the analyzer and `datadog-ci`.
```

**Comment by estherk15** [comment]
@juli1 Closing this in favor of https://github.com/DataDog/documentation/pull/24881/

**Pull request closed without merging**